qiita_pet/support_files/doc/source/analyzingsamples/index.rst

```diff
@@ -512,3 +512,526 @@ Beta Correlation
   * **Pearson** :ref:`[56]<reference56>` : Measures how strong the linear relationship is between 2 variables
 
 * Gives scatterplot of the distance matrix on the x-axis and the variable being tested on the y-axis
+
+-------------------------------------------------------------------------------
+
+Statistical Analysis to Justify Clinical Trial Sample Size Tutorial
```

Comment:
It also will be nice to have a small intro about what we are gonna cover (basic download and data processing) + an intro to the two examples and links to jump to them.

Reply:
Thank you for the feedback.
In terms of links to the examples:
Currently the first section, which is the only one of the three to consider both examples is sub-divided by method (Qiita redbiom plugin vs. command line redbiom) rather than example. I have written the introduction (see latest commit) to now reflect this, but am happy to reformat if you think this would be best.